scripts/bash/get-zs-p-ws.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo $(grep 'zone-cycles/wallsecond = ' ${1} | cut -d '=' -f 2 | xargs)
```

Comment:
I don't see what this script is used for.

Reply:
I ended up not needing it, but left it in because I think it's convenient. If you have the stdout from a phoebus (or any parthenon) run in a text file, it will pull out the zone-cycles per wallsecond. E.g.,
```bash
# prints something like 5.23e7
echo $(bash get-zs-p-ws.sh torus.out)
```